evaluation/saturatedTransactions.py

```diff
@@ -116,10 +119,14 @@ def run_experiment(peers: dict, number_of_transactions: int):
 
 
 def check_from_peers(submitted, confirmed, peers, peerList):
-    intersectionA = peers[peerList[0]].inter
     for tx in submitted:
-        if intersectionA.get_tx(tx[1]) == tx[1].value:
-            confirmed[tx] = tx
+        for peer in peerList:

```

Comment:
If the transaction value was found, that means it's been confirmed. Add it to confirmed.